src/components/Navbar.jsx

```diff
@@ -89,23 +89,32 @@ const CenterNav = ({ makerRunning, cjRunning, onClick }) => {
             'center-nav-link nav-link d-flex align-items-center justify-content-center' + (isActive ? ' active' : '')
           }
         >
-          {t('navbar.tab_send')}
+          <div className="d-flex align-items-start">
+            {t('navbar.tab_send')}
+            <TabActivityIndicator isOn={singleCollaborativeTransactionRunning} />
```

Comment:
Can you give this a 3px left margin? Just to move it away from the "d" of "Send" a bit. Cf.:

<img width="100" alt="Screenshot 2022-08-11 at 15 04 39" src="https://user-images.githubusercontent.com/10026790/184139878-1e72a7fd-d1b2-42d1-9fca-b15de5155bed.png">
<img width="78" alt="Screenshot 2022-08-11 at 15 04 43" src="https://user-images.githubusercontent.com/10026790/184139885-ab6e7be7-1a3a-4ecd-9377-6596f12e2676.png">

Reply:
I added spacing to all tab indicators.. as it will look different in other languages.. acceptable?

Reply:
Oh ok that's a good point. Hmm. I'm tempted to say let's optimize for English. The other languages will look okish but I bet 95% will use English anyway.

Reply:
Okay. Added class name "ms-1" instead of 3px to the Send link.. hope thats okay.

Maybe we can come up with a more generic indicator that looks good for all languages.

Reply:
Yep, we'll come up with something different anyway: https://github.com/joinmarket-webui/jam/pull/463#issuecomment-1212111322

Reply:
ms-1 is a bit much but let's leave it like that for now.